nodes/core/io/05-tls.html

```diff
@@ -61,13 +86,85 @@
                 return currentCert === '' || v != '';
             }},
             ca: {value:""},
+            certname: {value:""},
+            keyname: {value:""},
+            caname: {value:""},
+            uselocalfiles: {value:true},
```

Comment:
Unless there a reason not to, I'd prefer we infer this from which of `ca`/`caname` etc are set rather than add an explicit property to the node.